app/src/main/java/com/nguyennhatminh614/marvelapp/screen/comic/DetailComicFragment.kt

```diff
@@ -38,29 +56,39 @@ class DetailComicFragment :
 
             viewBinding.textDetailSeries.text = it.seriesDetail?.textDescription
 
-            viewBinding.textDetailSeries.setOnClickListener {
-                /* TODO implement later */
+            viewBinding.textDetailSeries.apply {
+                setOnClickListener { view ->
+                    if (!it.seriesDetail?.resourceUrl.isNullOrEmpty()) {
```

Comment:
```suggestion
                    if (it.seriesDetail?.resourceUrl.isNullOrEmpty().not()) {
```
dùng mấy cái extension này cho xịn e